### DISCUSSION TITLE
Does rome support vue sfc file out of box?

### DISCUSSION THREAD

**Discussion by lovetingyuan** [rome/tools#3608] [Q&A] [answered]
Does rome support vue sfc file out of box?

`rome check/format src/*.vue` ?

**Comment by MichaReiser** [comment, marked as answer]
No. Rome doesn't support `.vue` files yet. But we plan to add support for more languages in the next months

**Comment by skiss9** [comment]
I just tried to migrate a basic typescript vue app to rome, but running into that issue as well. It's not linting or formatting inside the .vue file. For the JS / TS part it's probably pretty easy to add support, it would require looking for .vue files and then processing what's inside the <script> tag
